scss/4-elements/_buttons.scss

```diff
@@ -111,20 +113,20 @@ $button-shadow-size: 4px;
  */
 .cads-button__tertiary {
   font-weight: $cads-font-weight__regular;
-  color: $cads-language__primary-button-colour;
+  color: $cads-language__button__colour--primary;
   background-color: $cads-palette__white;
-  border: solid $cads-border-width-small $cads-language__primary-button-colour;
+  border: solid $cads-border-width-small $cads-language__button__colour--primary;
   box-shadow: none;
 
   &:focus {
     box-shadow: 0 ($button-shadow-size - $cads-border-width-small) 0 0
-      $cads-language__focus-border-colour;
+      $cads-language__text__colour--secondary;
```

Comment:
Not sure about using a variable called `$cads-language__text__colour--secondary` to colour a box shadow - what if we want to theme them differently? Will the box shadow text colour always be the text colour or is it coincidence?